model/ftn/w3wavemd.ftn

```diff
@@ -2445,6 +2467,53 @@
                 END IF
 !
               END DO
+
+
+!/ CHECKPOINT
```

Comment:
The !/ is our switch identifier, these comments appear as switches with the '/' after.  Maybe we should remove it in comments?

Reply:
@ jessica this will be corrected.